src/Sdk/DTWebApi/WebApi/TaskAttachment.cs

```diff
@@ -107,5 +107,6 @@ public class CoreAttachmentType
     public class ChecksAttachmentType
     {
         public static readonly String StepSummary = "Checks.Step.Summary";
+        public static readonly String ResultsStepSummary = "Checks.Results.Step.Summary";
```

Comment:
why do you need a different type?
I assume the object will send to a different service endpoint.